client-frameworks-support/testing-utilities/README.md

```diff
@@ -1,13 +1,29 @@
 ## Luigi Testing Utilities
 
-Luigi Testing Utilities is a set of auxiliary functions used to enhance the user experience while testing Luigi based microfrontends. The functions abstract away Luigi specific logic from the tester so that it is easier for them to test. 
+Luigi Testing Utilities is a set of auxiliary functions used to enhance the user experience while testing Luigi based micro frontends. The functions abstract away Luigi specific logic from the tester so that it is easier for them to mock and assert Luigi functionality. 
```

Comment:
```suggestion
The Luigi Testing Utilities are a set of auxiliary functions used to enhance the user experience while testing Luigi-based micro frontends. The functions abstract away Luigi-specific logic from the tester so that it is easier for them to mock and assert Luigi functionality. 
```